concrete/src/User/UserInfo.php

```diff
@@ -512,37 +564,16 @@ public function markValidated()
      */
     public function changePassword($newPassword)
```

Comment:
Let's call the `update` method: they now execute the same queries, and perform the same operations.